code/tables/0_summary_table.do

```diff
@@ -1,4 +1,63 @@
 ********************************************************************************
 *			    Summary Table Attempt     							  		   *
 ********************************************************************************
-*HELLO WORLD!!
+* HELLO WORLD!! copied and pasted from run_project
+* make sure to delete after integrating
+
+clear all
+clear mata
+set more off
+set emptycells drop // If the interaction term is empty, don't report it in the table, reduces memory and variable necessity 
+
+set matsize 11000
+set maxvar 32000
+
+
+**********************IMPORTANT****************************
+* Change the paths to match path on your computer locally *
+***********************************************************
+
+* MAKE SURE TO SET WD TO 'BA-DRAFT' LEVEL OF FILE ORGANIZATION
+global repository `c(pwd)' //returns current working directory
+
+* Set OS
+global os `c(os)'
+
+* --- start of table here --- *
+
+// what does this do?
```

Comment:
when you wanna drop something or change ur data without changing it permanently, preserve/restore lets you make those changes, and then restore the data to the way it way before the preserve statement 

also these things you can google